main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"URezL/api"
+)
+
+func main() {
+	router := mux.NewRouter()
+	router.HandleFunc("/linkCut", api.PostLinkCut).Methods(http.MethodPost)
+	router.HandleFunc("/customLink", api.PostCustomLink).Methods(http.MethodPost)
+	router.HandleFunc("/login", api.PostLogin).Methods(http.MethodPost)
+	router.HandleFunc("/register", api.PostRegister).Methods(http.MethodPost)
+	router.HandleFunc("/links", api.GetLinks).Methods(http.MethodGet)
+	router.HandleFunc("/deleteLink", api.PostDeleteLink).Methods(http.MethodPost)
+	router.HandleFunc("/changeAddress", api.PostChangeAddress).Methods(http.MethodPost)
```

Comment:
Please consider using either PUT or PATCH verb here. Again, this is a modifying behaviour for "/links".